ts/WoltLabSuite/Core/Bootstrap.ts

```diff
@@ -176,6 +176,10 @@ export function setup(options: BoostrapOptions): void {
     void import("./Component/File/Upload").then(({ setup }) => setup());
   });
 
+  whenFirstSeen("[data-fancybox]", () => {
+    void import("./Component/Image/Viewer").then(({ setup }) => setup());
+  });
```

Comment:
We should support the legacy `.jsImageViewer` class for backwards compatibility while printing deprecation notices in the console.